drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/binary/downloading/source/PhantomJSGitHubBitbucketSource.java

```diff
@@ -20,32 +18,44 @@
 
     private static String BASE_DOWNLOAD_URL = "https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-";
 
+    private static String lastPhantomJSRelease = "2.1.1";
+
     public PhantomJSGitHubBitbucketSource(HttpClient httpClient, GitHubLastUpdateCache gitHubLastUpdateCache) {
```

Comment:
how about this: https://github.com/arquillian/arquillian-extension-drone/pull/88/commits/fc78f3c01ef916b13e3a1ade15b66c491e2fbe2c#diff-00a1f71655dc18c79ab7c7f8c0735aa3R180